tests/integration/mounts.bats

```diff
@@ -66,15 +66,18 @@ function teardown() {
 
 # https://github.com/opencontainers/runc/security/advisories/GHSA-m8cg-xc2p-r3fc
 @test "runc run [ro /sys/fs/cgroup mount]" {
-	# With cgroup namespace
-	update_config '.process.args |= ["sh", "-euc", "for f in `grep /sys/fs/cgroup /proc/mounts | awk \"{print \\\\$2}\"| uniq`; do grep -w $f /proc/mounts | tail -n1; done"]'
+	# Without cgroup namespace.
+	# shellcheck disable=SC2016
+	update_config '	  .linux.namespaces -= [{"type": "cgroup"}]
+			| .process.args |= ["sh", "-euc", "for f in `grep /sys/fs/cgroup /proc/mounts | awk \"{print \\\\$2}\"| uniq`; do test -e $f && grep -w $f /proc/mounts | tail -n1; done"]'
 	runc run test_busybox
 	[ "$status" -eq 0 ]
 	[ "${#lines[@]}" -ne 0 ]
 	for line in "${lines[@]}"; do [[ "${line}" == *'ro,'* ]]; done
 
-	# Without cgroup namespace
-	update_config '.linux.namespaces -= [{"type": "cgroup"}]'
+	# With cgroup namespace, if available.
+	[ ! -e "/proc/self/ns/cgroup" ] && return
```

Comment:
Seems like this test should be split, with one test having `requires cgroupns`. Otherwise we'll have a test where pass might mean only half of the test was actually run.

Reply:
This is fine with me, because there's absolutely no way to run the other half of the test anyway. The only difference is we won't know if the second part was run or not.

Nevertheless, let me split this.

Reply:
done; alas in the process I was not able to have such a fine grained ("one fix -- one commit") patch series.